addons/website/static/src/scss/website.scss

```diff
@@ -1666,6 +1666,7 @@ header {
         visibility: hidden;
         margin-top: $o-mega-menu-nav-height !important;
         border: none;
+        height: calc(100dvh - #{$o-mega-menu-nav-height});
```

Comment:
@nle-odoo You can probably close this since it was apparently fixed forcing 100vh by @bvr-odoo @SergeBayet at https://github.com/odoo/odoo/commit/579160b08c6e73d809be79f561214118f1284532 ... is using calc and dvh better?

TBH I think both solution are wrong :grimacing: Combining two elements with fixed positoin one inside another, margin-top, ... etc, it feels hacky. I think it would be better and lead to less code to:
- Make the offcanvas-body position: relative
- Make this position: absolute instead of fixed, and `inset: 0 0 0 0` = I want to cover the offcanvas-body, and in then make sense they are inside one another from the DOM point of view
- No margin-top, no height, max-height: none

I did not test but pretty sure it's the way to go. @bvr-odoo @SergeBayet thought?

Reply:
@nle-odoo You are right my fix is not good.. The calc is the good solution 👍🏼

Reply:
thanks

I updated the commit and commit message 👍